app/models/data.py

```diff
@@ -117,6 +112,8 @@ class State(db.Model, DataMixin):
     pui = db.Column(db.String)
     covidTrackingProjectPreferredTotalTestUnits = db.Column(db.String)
     covidTrackingProjectPreferredTotalTestField = db.Column(db.String)
+    totalTestResultsField = db.Column(db.String)
```

Comment:
sorry if I'm missing this - what is the `totalTestResultsField` attribute doing? I think you're only using the `totalTestResultsFieldDbColumn` right?

Reply:
It's doing absolutely nothing. Theo wanted it included in the DB since it's on the sheet.